packages/client/src/config.schema.json

```diff
@@ -204,6 +204,18 @@
                         "networkConnectivityTimeout": {
                             "type": "number",
                             "default": 10000
+                        },
+                        "websocketServerEnableTls": {
+                            "type": "boolean",
+                            "default": true
+                        },
+                        "autocertifierUrl": {
+                            "type": "string",
+                            "default": "https://ns1.fe6a54d8-8d6f-4743-890d-e9ecd680a4c7.xyz:59833"
```

Comment:
Is this our real production value?

Reply:
This where the AutoCertifier is deployed now and likely for when we first deploy the changes